doc/man1/openssl-ocsp.pod.in

```diff
@@ -326,6 +310,10 @@ specified in the B<-rsigner> option is used.
 The private key password source. For more information about the format of I<arg>
 see L<openssl(1)/Pass Phrase Options>.
 
+=item B<-rother> I<file>
+
+Additional certificates to include in the OCSP response.
+
```

Comment:
Note: Some of the items in the OPTIONS section where moved around to match the order in the SYNOPSIS section.